src/tools/ilasm/ilasm.slnx

```diff
@@ -0,0 +1,4 @@
+<Solution>
+  <Project Path="ILAssembler.Tests/ILAssembler.Tests.csproj" />
```

Comment:
Should we use `src` / `tests` directory structure that's common in the repo? Both illink and StressLogAnalyzer that are siblings of this directory are on the src / tests plan.